DSharpPlus/DiscordClient.cs

```diff
@@ -2438,6 +2447,19 @@ internal async Task SendIdentifyAsync(StatusUpdate status)
             };
             var payloadstr = JsonConvert.SerializeObject(payload);
             await this._webSocketClient.SendMessageAsync(payloadstr).ConfigureAwait(false);
+
+            if (this.Configuration.Intents.HasValue)
+            {
+                var intentList = Enum.GetValues(typeof(DiscordIntents)).Cast<DiscordIntents>().ToArray();
+                for (int i = 0; i < intentList.Length; ++i)
```

Comment:
```cs
var intentList = Enum.GetValues(typeof(DiscordIntents)).Cast<DiscordIntents>();
foreach( var intent in intentList)
```
Potential simplification.

Reply:
True. I had a for loop being faster in mind, but in this case I suppose it doesn’t really make a difference.

Reply:
Agreed. I think the ToArray would remove any benefits for doing it as a for loop.

Reply:
The array from `Enum.GetValues()` can simply be casted here to `DiscordIntents[]`, so do that instead. 😉